src/main/java/net/sf/jabref/logic/autocompleter/NameFieldAutoCompleter.java

```diff
@@ -43,26 +47,27 @@
      * @see AutoCompleterFactory
      */
     NameFieldAutoCompleter(String fieldName, AutoCompletePreferences preferences) {
-        this(new String[] {fieldName}, false, preferences);
+        this(new String[] {Objects.requireNonNull(fieldName)}, false, preferences);
     }
 
     public NameFieldAutoCompleter(String[] fieldNames, boolean lastNameOnlyAndSeparationBySpace,
```

Comment:
Would it make sense to change the fieldNames array to a List<String> as well? If you only pass a list with one element, you can use Collections.singletonList("element");